src/main/java/uk/gov/digital/ho/hocs/document/JSONToSimpleTextConverter.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.digital.ho.hocs.document;
+
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.JsonNodeType;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.text.WordUtils;
+
+import java.io.IOException;
+
+@Slf4j
+public class JSONToSimpleTextConverter {
+    public static final String NEW_LINE_STR = "\n            ";
```

Comment:
Are these extra spaces intentional?

Reply:
Yeh, they're to align the left margin of the text.